clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -140,6 +140,34 @@ abstract class OpenstackClientProvider {
     //TODO: Handle heat autoscaling migration to senlin in versions > Mitaka
   }
 
+  /***
+   * Get a Spinnaker Server Group (Openstack Heat Stack).
+   * @param stackName
+   */
+  Stack getServerGroup(String stackName) {
+    try {
+      def stack = client.heat().stacks().getStackByName(stackName)
```

Comment:
should support regions: `client.useRegion()` (apparently a refactor to have a getRegionClient() is forthcoming, but it hasn't been opened a PR for)